Web/Assignment1/YourProfile.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+	<html>
+		<head>
+			<link rel="stylesheet" href="style1.css" type="text/css">
+		</head>
+		<body>
+			<div class="wrapper">
+				<div class="nav">
```

Comment:
ACTION: didn't find class for nav, nav-left, nav-right in style1.css. Please check css class you are using in your html file is present in css file.